### DISCUSSION TITLE
【二次开发申请】将本项目功能复刻至Android项目子功能

### DISCUSSION THREAD

**Discussion by 1250422131** [YunYouJun/cook#53] [Q&A]
### Clear and concise description of the problem

是这样的，我是一名安卓原生开发者，在github上发现了[cook](https://cook.yunyoujun.cn/)

我最近正在开发一款APP，叫做食选，它目前计划了两个功能，其中一个功能就来自本项目。

其一是复刻了[cook](https://cook.yunyoujun.cn/)通过选择食材来得到食物结构，并且在其基础上扩展一些功能。

其二则是由用户录入食堂的菜品，随机选择一个给用户。

最终，通过AI模型推算出用户下一次可能吃的食物和类似食物。

### Suggested solution

其中第一个功能采用了你的开源项目收集的数据，想问一下能否允许我进行数据复用的行为？

我会注明来源，并且鼓励用户在原项目投稿菜品。

项目地址：https://github.com/1250422131/FoodChoice

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by YunYouJun** [comment]
您好，数据可以自由使用，不过希望可以注明出处，如果您对数据集进行了些修改也可注明，也期待您可以提交 PR 补充新数据。
